libgalois/include/katana/PropertyGraph.h

```diff
@@ -767,6 +804,9 @@ CreateSymmetricGraph(PropertyGraph* pg);
 KATANA_EXPORT Result<std::unique_ptr<PropertyGraph>>
 CreateTransposeGraphTopology(const GraphTopology& topology);
 
+/// Report approximate memory use for a property view
+KATANA_EXPORT uint64_t ApproxMemUse(const PropertyGraph::PropertyView& pview);
+
```

Comment:
Why not make this a method on PropertyView.

Reply:
Oh, good idea, thanks.